src/delta_lake/delta_service.py

```diff
@@ -19,10 +21,11 @@
 logger = logging.getLogger(__name__)
 
 MAX_SAMPLE_ROWS = 1000
+CACHE_EXPIRY_SECONDS = 3600  # Cache results for 1 hour by default
```

Comment:
I can see this causing confusion in the future when some one updates data in a table and doesn't see the results for an hour because of the cache. It might be worth having a discussion at the standup

Reply:
I chose a longer duration because this is for the MCP service, and I don’t expect the data table to be updated frequently. In most cases, our data tables are created once and remain unchanged thereafter.